src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -850,6 +863,52 @@ void CC32xxWiFi::sock_event_handler(void *context)
     }
 }
 
+/*
+ * CC32xxWiFi::http_server_callback()
+ */
+void CC32xxWiFi::http_server_callback(void *context1, void *context2)
+{
+    if(!context1 || !context2)
+    {
+        return;
+    }
+
+    SlHttpServerEvent_t *event = static_cast<SlHttpServerEvent_t *>(context1);
+    SlHttpServerResponse_t *response = static_cast<SlHttpServerResponse_t *>(context2);
+
+    switch (event->Event)
+    {
+        case SL_NETAPP_HTTPGETTOKENVALUE_EVENT:
+        {
+            unsigned char *ptr;
+
+            ptr = response->ResponseData.token_value.data;
+            response->ResponseData.token_value.len = 0;
+
+            for (unsigned i = 0; i < httpGetCallbacks_.size(); ++i)
+            {
+                if (memcmp(event->EventData.httpTokenName.data,
```

Comment:
Are you purposefully allowing prefix matches on tokens? I think that's a good idea, but then we also need to send the actual token value to the callback function.